src/main/java/org/gridsuite/modification/server/RestResponseEntityExceptionHandler.java

```diff
@@ -20,14 +20,13 @@ public class RestResponseEntityExceptionHandler {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(RestResponseEntityExceptionHandler.class);
 
-    @ExceptionHandler(value = {NetworkModificationException.class})
-    protected ResponseEntity<Object> handleException(RuntimeException exception) {
+    @ExceptionHandler(NetworkModificationException.class)
+    protected ResponseEntity<Object> handleNetworkModificationException(NetworkModificationException exception) {
```

Comment:
I agree

Reply:
That's what I did firstly, but it break lot of tests, because the API answers are different. 
I supposed that it's on purpose, that we don't want to return every exceptions but only few ones.

Reply:
ok